packages/Manifest.yml

```diff
@@ -0,0 +1,119 @@
+---
+global:
+  version: 0.0.1
```

Comment:
This can be retrieved from the current git tag

You will need to delay the current CI trigger to be run on "git tag" instead of "on merge on master"

Reply:
Ok I'll change that 👍